src/main/java/com/commercetools/sync/commons/enums/Error.java

```diff
@@ -4,15 +4,26 @@
 import javax.annotation.Nonnull;
 
 /**
- * Represents all error messages across the sync update action build modules.
+ * Represents all error messages across the sync modules.
  */
 public enum Error {
```

Comment:
Actually, when I come to think of it again, it's actually better to use constant `Strings` instead of Enums in this case. Because most of them are _format_ `Strings` and if they are used as enums (`ENUM.getDescription()`) then the IDE will not be able to show warnings if the number of arguments don't match as opposed to directly using the constant String. 

Also it's not the best idea to keep all strings in one class (enum in this case), so if it get's really big and confusing. So I refactored them to be at the Utils where they are only needed. 	

29a1098